src/Processors/Formats/Impl/ParquetBlockInputFormat.h

```diff
@@ -288,6 +293,9 @@ class ParquetBlockInputFormat : public IInputFormat
     std::exception_ptr background_exception = nullptr;
     std::atomic<int> is_stopped{0};
     bool is_initialized = false;
+    String metadata_cache_key;
+    bool use_metadata_cache = false;
+    UInt64 metadata_cache_max_entries{0};
```

Comment:
Nitpick: consider grouping this into something like `struct Cache...`

Reply:
Done.